microprofile/WFLY-13089-microprofile-rest-client-1.4.adoc

```diff
@@ -0,0 +1,54 @@
+= EAP7-1429 / WFLY-13089 Microprofile REST Client 1.4 support
+:author:            Alessio Soldano
+:email:             asoldano@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
```

Comment:
JFYI: The URL is now being redirected to https://issues.redhat.com/browse

Reply:
Right, I copy-pasted the headers section from an old RFE...